### DISCUSSION TITLE
Next steps

### DISCUSSION THREAD

**Discussion by diogomqbm** [t10d/cra-template-kingdom#28] [Ideas]
Hey

I've been thinking about our next steps with this project and I think we should explore the logged-in user interface. I think we should build:
- Profile screen
- Navbar
- Dashboard
- Users list

What do you think about these features for our v2?

**Comment by youphenrique** [comment]
For me, I would prioritize a Navbar/Header, Sidebar (menu) and that Dashboard page (as an inittal page) for this v2.
